spec/rails_helper.rb

```diff
@@ -50,7 +50,7 @@ def with_settings(settings, temp_values)
   end
 end
 
-VCR::MATCH_EVERYTHING = { match_requests_on: %i[method uri headers body] }.freeze
+VCR::MATCH_EVERYTHING = { match_requests_on: %i[method uri body] }.freeze
```

Comment:
What's the rationale here?

Reply:
Good to know! will update

Reply:
I would not like to blanket start ignoring headers in VCR - there are several integrations where they are significant for identifying users, sessions, etc.

Reply:
For what it's worth, for your specific case you can just use `{ match_requests_on: %i[method uri body] }` in place of `VCR::MATCH_EVERYTHING` and it should do the same thing. I would just avoid modifying this specific constant to mean something besides "EVERYTHING".

Reply:
😛that definitely makes sense — should have thought that one through.  Thanks for the feedback!